aioamqp/exceptions.py

```diff
@@ -49,3 +49,13 @@ def __init__(self, delivery_tag):
     def __repr__(self):
         return 'Publish failed because a nack was received for delivery_tag {}'.format(
             self.delivery_tag)
+
+
+class PublishReturned(AioamqpException):
+    def __init__(self, reply_code, reply_text, exchange, routing_key):
```

Comment:
I think @dzen is talking about the constructor implementation which in this case is not necessary. 
And if he is not: the constructor is not necessary :)